test/configCases/trusted-types/continue-on-policy-creation-failure/index.js

```diff
@@ -0,0 +1,36 @@
+it("can continue on policy creation failure", function () {
+	// emulate trusted types in a window object
+	window.trustedTypes = {
+		createPolicy: () => {
+			throw new Error("Rejecting createPolicy call");
+		}
+	};
+
+	const createPolicySpy = jest.spyOn(window.trustedTypes, "createPolicy");
+	const consoleWarn = jest.spyOn(console, "warn").mockImplementation(() => {});
```

Comment:
I mean please create a real test case (as you describe in your PR description), `spy` is like unit testing, that is not good

Reply:
@alexander-akait , I'm not sure that there's a way to add a check-in-able test case -- in order to see the effect, you need to have a website running and serving CSP headers. But I can work on making a simple repro app, I'll try to do that later today or tomorrow.

Reply:
Yeah thank you